tests/unit/test-cmd/test.run.js

```diff
@@ -477,6 +479,28 @@ describe('run', () => {
         });
     });
 
+    it('pauses the web-ext process (CTRL+Z in shell console)', () => {
+      const {reloadStrategy, kill} = prepare();
+      const fakeStdin = new tty.ReadStream();
+      const setRawMode = sinon.spy(fakeStdin, 'setRawMode');
+
+      return reloadStrategy({}, {stdin: fakeStdin})
+        .then(() => {
+          fakeStdin.emit('keypress', 'z', {name: 'z', ctrl: true});
+        }).then(() => {
+          assert.ok(kill.called);
+          assert.deepEqual(kill.firstCall.args, [process.pid, 'SIGTSTP']);
+          sinon.assert.callOrder(setRawMode, setRawMode, kill, setRawMode);
+          sinon.assert.calledThrice(setRawMode);
+          assert.equal(setRawMode.firstCall.args[0], true);
+          assert.equal(setRawMode.secondCall.args[0], false);
+          assert.equal(setRawMode.lastCall.args[0], true);
+        })
+        .then(() => {
+          fakeStdin.emit('keypress', 'c', {name: 'c', ctrl: true});
```

Comment:
@rpl good call. i added those to all cases that test have the loop, in other cases for this function since `stdin` is not `TTY` they shouldn't enter the loop in the first place.